hail/python/hailtop/aiocloud/aiogoogle/client/storage_client.py

```diff
@@ -132,7 +132,7 @@ async def __aexit__(self,
 
 
 class ResumableInsertObjectStream(WritableStream):
-    def __init__(self, session: GoogleSession, session_url: str, chunk_size: int):
+    def __init__(self, session: BaseSession, session_url: str, chunk_size: int):
```

Comment:
What's the reason for this change? This is the Google storage client, I feel like it needs a Google session

Reply:
I didn't kill GoogleStorageClient b/c I had to add like five lines to each client.